backend/src/paymaster/pimlico.ts

```diff
@@ -182,7 +199,7 @@ export async function getERC20Paymaster(
             nativeAssetOracle: ORACLE_ADDRESS[chainId][NATIVE_ASSET[chainId]],
             tokenAddress: TOKEN_ADDRESS[chainId][erc20],
             tokenOracle: ORACLE_ADDRESS[chainId][erc20],
-            owner: "0x4337000c2828f5260d8921fd25829f606b9e8680" // pimlico address
+            owner: "0x4337000c2828F5260d8921fD25829F606b9E8680" // pimlico address
```

Comment:
Why is this hardcoded here? What if we wish to change it to our own tokens?

Reply:
this is to generate the pimlico address which on this version wont be using that since I have included those in constants file itself. Now its just used for unit tests